### ISSUE TITLE
Start fail with another ['postgresql']['dir']

### ISSUE THREAD

**Issue by tiago-cruz-movile** [hw-cookbooks/postgresql#260]
If you change the directory the daemon does not start 'cause still lookin files on /var/lib/pgsql:

```
[root@pgsql ~]# cat /zfs-postgres/pgsql/9.4/pgstartup.log
LOG:  skipping missing configuration file "/var/lib/pgsql/data/postgresql.auto.conf"
FATAL:  data directory "/var/lib/pgsql/data" does not exist
```
Packages used:

```
postgresql94-devel-9.4.1-1PGDG.rhel6.x86_64
postgresql94-libs-9.4.1-1PGDG.rhel6.x86_64
postgresql94-9.4.1-1PGDG.rhel6.x86_64
postgresql94-server-9.4.1-1PGDG.rhel6.x86_64
```

**Comment by slyness** [comment]
I'm actively working on an update that will address this issue.

**Comment by slyness** [comment]
Soonicorn. #314

**Comment by slyness** [comment]
Resolved as of release 4.0.0 - https://github.com/hw-cookbooks/postgresql/releases/tag/v4.0.0
